datahub-graphql-core/src/main/resources/timeline.graphql

```diff
@@ -0,0 +1,146 @@
+extend type Query {
+    """
+    Get list of schema blame
+    """
+    listSchemaBlame(input: ListSchemaBlameInput!): ListSchemaBlameResult
```

Comment:
The name of this makes me think that we are getting a _list_ of different versions of a schema. But actually it seems like we are just getting 1 semantic schema version.

For example, if we are going to keep the name "Blame" then it feels like this should be something like:

getSchemaBlame(urn, version)

Reply:
Resolved offline